pkg/player/player.go

```diff
@@ -1,26 +1,28 @@
 package player
 
 import (
-	"errors"
+	"fmt"
 	"net/http"
 	"time"
 )
 
 const endpoint = "https://api.spotify.com/v1/me/player/pause"
 
 func Pause(client *http.Client, d time.Duration) error {
+	timer := time.NewTimer(d)
+	<-timer.C
+
 	req, err := http.NewRequest(http.MethodPut, endpoint, nil)
 	if err != nil {
 		return err
 	}
-	timer := time.NewTimer(d)
-	<-timer.C
 	resp, err := client.Do(req)
 	if err != nil {
 		return err
 	}
-	if resp.StatusCode != 204 {
-		return errors.New("Status Code Error: response status code is " + string(resp.StatusCode))
+	if resp.StatusCode != http.StatusNoContent {
+		return fmt.Errorf("Status Code Error: %d", resp.StatusCode)
```

Comment:
if use variables when creating error, u can use `fmt.Errorf`. It support a format style.